core/src/components/cat-dropdown/cat-dropdown.scss

```diff
@@ -15,49 +15,83 @@ $-max-height: calc(100vh - 48px);
 }
 
 ::slotted(nav) {
-  padding-top: $cat-body-margin-bottom * 0.5;
-  padding-bottom: $cat-body-margin-bottom * 0.5;
+  padding: cat-token('size.spacing.xs');
```

Comment:
spacing values are deprecated as tokens. they only exist to generate the correct spacing classes. Use hard-coded spacings here.

Reply:
Oh I was not aware of that. Do we not want to have spacing tokens anymore?